doc/commercial-services.md

```diff
@@ -4,6 +4,11 @@ There is a bunch of services that offer integration with your existing CI pipeli
 reports. Please note these are not associated with the SimpleCov project itself, so please report problems with
 these integrations with their respective owners.
 
+#### [Codacy](https://github.com/codacy/codacy-coverage-reporter)
+*by [Codacy](https://www.codacy.com/)*
+
+Upload coverage reports to [codacy.com](https://www.codacy.com/), a hosted (or self-hosted) software quality analysis and that also includes coverage reporting.
```

Comment:
I feel like there's a `tool` missing somewhere in here. Like instead of "and" I guess :)

Reply:
What do you think of this alternative? Also, do you want me to change codeclimate's description (it has the same wording)
```suggestion
Upload coverage reports to [codacy.com](https://www.codacy.com/), a hosted (or self-hosted) software quality analysis platform that also includes coverage reporting.
```

Reply:
ah, yes please!